lib/shell/shellscript.js

```diff
@@ -74,6 +74,18 @@ class ShellScriptWorker {
             } else {
                 console.log('No content type information file found');
             }
+
+            if (await fs.pathExists(preparedFiles.optionsFile)) {
+                try {
+                    const options = await fs.readJSON(preparedFiles.optionsFile);
+                    if (options.postProcess === true || options.postProcess === "true") {
+                        rendition.postProcess = true;
```

Comment:
```suggestion
                        rendition.postProcess = (options.postProcess === true || options.postProcess === "true");
```
And remove the `if`condition.